### PR TITLE
Remove alpha tag from psammead-timestamp-container

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves #NUMBER

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/containers/psammead-timestamp-container/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 5.0.8 | [PR#?](https://github.com/bbc/psammead/pull/?) Remove alpha tag from package.json |
```
Comment (simonsinclair) [inline packages/containers/psammead-timestamp-container/CHANGELOG.md]:
```suggestion
| 5.0.8 | [PR#3935](https://github.com/bbc/psammead/pull/3935) Remove alpha tag from package.json |
```

**Review by simonsinclair — APPROVED** [review]
🥳

**Comment by RichardPK** [comment]
💩

**Review by ryanmccombe — APPROVED** [review]
💩

**Pull request merged**